source/browseMode.py

```diff
@@ -7,6 +7,7 @@
 from typing import (
 	Any,
 	Callable,
+	Generator,

```

Comment:
We are on Python 3.11, so this should use `from collections.abc import Generator`

Reply:
[Here](https://www.phind.com/search?cache=q2x75q8negs9q9yv5zjrh8tx) they say `collections.abc.Generator` use for type hinting is deprecated in favor of `typing.Generator`. And I havent found any usage of `abc.Generator` as type hint in NVDA. Do you have any evidence to support your point of view?

Reply:
The docs themselves say the deprecation is the other way around - that `typing.Generator` is the deprecated alias
https://docs.python.org/3/library/typing.html#typing.Generator